tests/boardfarm_plugins/boardfarm_sample_plugin/prplmesh_config.json

```diff
@@ -0,0 +1,49 @@
+{
+    "prplmesh_dummy": {
+        "name": "gateway",
+        "board_type": "prplmesh_docker",
+        "role": "controller",
+        "delay": 7,
+        "conn_cmd": "",
+        "cleanup_cmd": "docker container rm -f gateway",
+        "connection_type": "local_cmd",
+        "devices": [
+            {
+                "name": "repeater1",
```

Comment:
Although the README claims that devices have to be declared, is that really true?

Reply:
> Although the README claims that devices have to be declared, is that really true?

With no devices boardfarm will raise exception, so we need specify at least one device.

Reply:
> Although the README claims that devices have to be declared, is that really true?

I've rechecked that option - we can specify empty device list `"devices": [],` and that actually works, but `"devices": []` list, even if it's empty, must be present anyway
Added note about that to README.